includes/class-wc-stripe.php

```
@@ -253,6 +253,9 @@ public function init() {
 
 			$checkout_sessions_ajax_handler = new WC_Stripe_Checkout_Sessions_Ajax_Handler();
 			$checkout_sessions_ajax_handler->init_hooks();
+
+			$checkout_session_lifecycle = new WC_Stripe_Checkout_Session_Lifecycle();
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '257', column '38'). (undefined)

(MissingImport)